package.json

```diff
@@ -85,5 +85,8 @@
   "resolutions": {
     "@lerna/conventional-commits": "https://github.com/ntucker/lerna-conventional-commits.git"
   },
-  "version": "0.0.0"
+  "version": "0.0.0",
+  "dependencies": {
+    "downlevel-dts": "^0.7.0"
```

Comment:
why is this a runtime dep, that non-ts users would need to download too?

Reply:
in this case it doesn't matter because this is the top level package - which is just for workspaces (doesn't get published); however, i'll move it for bookkeeping